pkg/auth/model/model.go

```diff
@@ -7,16 +7,62 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/treeverse/lakefs/pkg/auth/crypt"
+
+	"github.com/go-openapi/swag"
 	"github.com/treeverse/lakefs/pkg/kv"
 	"golang.org/x/crypto/bcrypt"
+	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 const (
-	StatementEffectAllow = "allow"
-	StatementEffectDeny  = "deny"
+	StatementEffectAllow   = "allow"
+	StatementEffectDeny    = "deny"
+	PartitionKey           = "auth"
+	groupsPrefix           = "groups"
+	groupsUsersPrefix      = "groups_users"
```

Comment:
This is great, but I think we still have a problem with prefix scanning.
Maybe change to "gPolicies, "gUsers", "uPolicies" "uCredentials"

Reply:
done